code/modules/hydroponics/seed_packets.dm

```diff
@@ -322,3 +322,6 @@ GLOBAL_LIST_BOILERPLATE(all_seed_packs, /obj/item/seeds)
 
 /obj/item/seeds/rose/blood
 	seed_type = "bloodrose"
+
+/obj/item/seeds/rose/gnomes
```

Comment:
Why is seed packet a subtype of rose seed packet?

Reply:
I have no clue why, but it generates as one ingame. I didn't make it a subtype in the seed datum, so my only guess is that code is borked somewhere.